### DISCUSSION TITLE
Who composed the music 🎼  for the ballet "The Nutcracker" ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#41] [Galaxy Qs] [answered]
Who composed the music for the ballet "The Nutcracker" ?

**Comment by lotus647** [comment]
Tchaikovsky

**Comment by opticaldrive** [comment, marked as answer]
Pyotr Tchaikovsky

**Reply by Sriansh-raj** [reply]
Congratulations @V205Arduino you just got a `Galaxy Brain` achievement. 🎉

Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃

**Reply by opticaldrive** [reply]
Will do!
Thank you!

**Reply by lotus647** [reply]
I said it before though... Oh well 🤷🏻

**Reply by opticaldrive** [reply]
Yes, You should rightfully get it.
Ask @Sriansh-raj ?

**Reply by Sriansh-raj** [reply]
Well @V205Arduino I can just mark only one comment as answer in one discussion.
And @theClawsmos when I saw the discussion, @V205Arduino had given the answer in full names so I marked his as the answer.
@theClawsmos you can answer in other discussions as well.

Hope you got this :)

~Thank you for your cooperation.

**Comment by ChinoBeach-Colgate** [comment]
original author and composer in 1816 E.T. A. Hoffmann

**Comment by NomanGhiasy** [comment]
George Balanchine

**Reply by lotus647** [reply]
I feel like this is subjective, because people have made remakes, and differences, etc.

**Comment by saurabhcoded** [comment]
Pyotr Tchaikovsky
